pagopa/pn-frontend
# Feature/pn-10851 - web api call for Sender Dashboard

## Short description
This PR also includes changes made for pn-data-viz library configuration (Feature/pn-10451)

## List of changes proposed in this pull request
- add types for echarts
- add default theme for PnEcharts
- add route for sender dashboard page (Statistics.page.tsx)
- add FiledNotificationsStatistics component to test PnEcharts
- change Statistics.page.ts 
- add Statistics menu item
- add getStatistics action to start web api call
- add getStatistics api call and the relative mock to simulate the server response
- change statistics/reducers.ts to manage and store the response

## How to test
- login into pa-webapp
- verify the Statistics menu item is available and works (follow the route on click)
- verity the FiledNotificationsStatistics renders properly on the destination page
- using Redux DevTools verify the "statistics" state got filled with mock data (mocked api call)
